### PR TITLE
Added Manhattan Distance

### PR DESCRIPTION
**Pull request by LegendaryGene**

### CODE REVIEW

File: furthermaths/distance.py [hunk 1]
```diff
@@ -20,6 +20,33 @@ def dist(*args):
...
+    
+def manhattan_dist(*args):
+    """
+    Calculates the manhattan distance between two points
+
+    :param args: the points
+    :return: manhattan distance
+    """
+    if len(args) not in [2, 4]:
+        raise ValueError("dist() takes 2 or 4 arguments")
```
Comment (ChrisO345) [inline furthermaths/distance.py:32]:
```suggestion
        raise ValueError("manhattan_dist() takes 2 or 4 arguments")
```

File: furthermaths/distance.py [hunk 2]
```diff
@@ -20,6 +20,33 @@ def dist(*args):
...
+    if type(args[0]) == set or type(args[0]) == list:
+        pos1 = args[0]
+        pos2 = args[1]
+        if len(pos1) == 2 and len(pos2) == 2:
+            x1, y1, x2, y2 = pos1[0], pos1[1], pos2[0], pos2[1]
+        else:
+            raise ValueError("dist() takes 2 sets with length 2")
```
Comment (ChrisO345) [inline furthermaths/distance.py:39]:
```suggestion
            raise ValueError("manhattan_dist() takes 2 sets with length 2")
```

File: furthermaths/distance.py [hunk 3]
```diff
@@ -20,6 +20,33 @@ def dist(*args):
...
+    else:
+        x1, y1, x2, y2 = args[0], args[1], args[2], args[3]
+    if x1>x2 and y1>y2:
+    	return (x1 - x2, y1 - y2)
+    elif x1<x2 and y1>y2:
+    	return (x2 - x1, y1 - y2)
+    elif x1>x2 and y1<y2:
+    	return (x1 - x2, y2 - y1)
+    else:
+    	return (x2 - x1, y2 - y1)
```
Comment (ChrisO345) [inline furthermaths/distance.py:49]:
```suggestion
    return abs(x1 - x2 + y1 - y2)
```

**Review by ChrisO345 — CHANGES_REQUESTED** [review]

**Comment by ChrisO345** [comment]
Thanks for doing this.

**Review by ChrisO345 — COMMENTED** [review]
Looks good now. Thanks

**Comment by LegendaryGene** [comment]
Cheers!

**Pull request merged**